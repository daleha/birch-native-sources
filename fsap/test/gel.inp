temp.gel.out
1
     23.7  0.4
     9.46  0.7
     6.75  0.85
     4.26  1.15
     2.26  1.6
     1.98  1.75
     0.58  2.7
     0 0
6
Test of GEL
pI230
2.7
pI259
2.95
pI286
2.35

0
