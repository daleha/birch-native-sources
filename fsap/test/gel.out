Test of GEL
   STD LEN      DIST  PRED LEN DEVIATION      %DEV      C[I]
     23.70     0.400     23.79    -0.086    -0.364     5.791
      9.46     0.700      9.28     0.178     1.876     5.905
      6.75     0.850      6.85    -0.096    -1.416     5.746
      4.26     1.150      4.21     0.045     1.058     5.855
      2.26     1.600      2.34    -0.082    -3.618     5.694
      1.98     1.750      1.95     0.025     1.272     5.851
      0.58     2.700      0.57     0.009     1.615     5.835
M0=     0.1723  L0=    -1.7282  CCAP=     5.8109
SC=     0.0720  SD=     0.1199

UNKNOWN FRAGMENTS:
  FRAGMENT  DISTANCE PREDICTED LENGTH
pI230           2.70            0.571
pI259           2.95            0.364
pI286           2.35            0.940

